src/platform/test.rs

```diff
@@ -10,16 +10,32 @@
 use platform::{self, OsIpcChannel, OsIpcReceiverSet};
 use platform::{OsIpcSharedMemory};
 use std::collections::HashMap;
+#[cfg(not(any(feature = "force-inprocess", target_os = "android")))]
+use std::process::{Command, Stdio};
 use std::sync::Arc;
 use std::time::{Duration, Instant};
 use std::thread;
+#[cfg(not(any(feature = "force-inprocess", target_os = "android")))]
```

Comment:
This is also needed for other tests to compile (or you need to compile out the `get_channel_name_arg` too at least).